clients/destination.go

```diff
@@ -30,6 +30,14 @@ func NewLocalDestinationClient(p plugins.DestinationPlugin) *DestinationClient {
 	}
 }
 
+func (c *DestinationClient) Name(ctx context.Context) (string, error) {
+	res, err := c.pbClient.GetExampleConfig(ctx, &pb.GetExampleConfig_Request{})
```

Comment:
Should we create a protobuf call just for that?

Reply:
Ok, will update 👍

Reply:
Done; the RPC calls have been split into three separate ones now (name, version, config)